com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-basic-unlit-structure.md

```diff
@@ -0,0 +1,176 @@
+# URP unlit basic shader
+
+This example shows a basic URP-compatible shader. This shader fills the mesh shape with a color predefined in the shader code.
+
+To see the shader in action, copy and paste the following ShadeLab code into the Shader asset.
+
+```c++
+// This shader fills the mesh shape with a color predefined in the code.
+Shader "Example/URPUnlitShaderBasic"
+{
+    // The properties block of the shader. In this example this block is empty since 
+    // the output color is predefined in the fragment shader code.
+    Properties
+    { }
+
+    // The SubShader block containing the Shader code. 
+    SubShader
+    {
+        // Tags define when and under which conditions a SubShader block or a pass
+        // is executed.
+        Tags { "RenderType" = "Opaque" "RenderPipeline" = "UniversalRenderPipeline" }
+        
+        Pass
+        {
+            // The HLSL code block. Unity SRP uses the HLSL language.
+            HLSLPROGRAM
+            // This line defines the name of the vertex shader. 
+            #pragma vertex vert
+            // This line defines the name of the fragment shader. 
+            #pragma fragment frag
+
+            // The Core.hlsl file contains definitions of frequently used HLSL macros and
+            // functions, and also contains #include references to other HLSL files 
+            // (for example, Common.hlsl, SpaceTransforms.hlsl, etc.).
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"            
+            
+            // The structure definition defines which variables it contains.
+            // This example uses the Attributes structure as an input structure in the
+            // vertex shader.
+            struct Attributes
+            {
+                // The positionOS variable contains the vertex positions.
+                float4 positionOS   : POSITION;                 
+            };
+
+            struct Varyings
+            {
+                // The positions in this struct must have the SV_POSITION semantic.
+                float4 positionHCS  : SV_POSITION;
+            };            
+
+            // The vertex shader definition with paroperties defined in the Varyings 
+            // structure. The type of the vert function must match the type (struct) that it
```

Comment:
When reading these lines on the doc page "that it" flows on to the the next line like this:
```
// The vertex shader definition with paroperties defined in the Varyings 
// structure. The type of the vert function must match the type (struct) t
hat it
// returns.
```
I think about 85 chars is the limit for each line

Reply:
Thanks, fixed this.